ml-covid19-backend/src/main/kotlin/com/covidproject/coviddatavisualizer/controller/CovidController.kt

```diff
@@ -0,0 +1,37 @@
+package com.covidproject.coviddatavisualizer.controller
+
+import com.covidproject.coviddatavisualizer.controller.models.SingleResponse
+import com.covidproject.coviddatavisualizer.service.CovidService
+import com.covidproject.coviddatavisualizer.service.models.CountWithDateResponse
+import org.springframework.format.annotation.DateTimeFormat
+import org.springframework.web.bind.annotation.*
+import java.util.*
+
+@RestController
+@RequestMapping("/covidInfo")
```

Comment:
```suggestion
@CrossOrigin("*")
@RestController
@RequestMapping("/covidInfo")
```
Добавь cross origin пожалуйста